underfs/swift/src/main/java/alluxio/underfs/swift/http/SwiftDirectClient.java

```diff
@@ -67,10 +68,10 @@ public static SwiftOutputStream put(Access access, String objectName) {
         httpCon.connect();
         return new SwiftOutputStream(httpCon);
       }
-      LOG.debug("Not an instance of HTTP URL Connection");
-    } catch (IOException e) {
+      throw new Exception("Not an instance of HTTP URL Connection");
+    } catch (Exception e) {
```

Comment:
It is unexpected. An HTTP url should return a HTTPURLConnection. I made the correction. Thanks.